site/content/contribute/more-info/mobile/developer-setup/_index.md

```diff
@@ -8,207 +8,250 @@ aliases:
 
-- SDK Tools (you may have to click "Show Package Details" to expand packages)
-    ![image](/img/mobile/sdk_tools.png)
-    - Android SDK Build-Tools 29.0.2
-    - Android Emulator
-    - Android SDK Platform-Tools
-    - Android SDK Tools
-    - Google Play services
-    - Intel x86 Emulator Accelerator (HAXM installer)
-    - Support Repository
-        -   Android Support Repository
-        -   Google Repository
+- SDK Tools (you may have to click "Show Package Details" to expand packages):
+  - Android SDK Build-Tools 29.0.2
+  - Android Emulator
+  - Android SDK Platform-Tools
+  - Android SDK Tools
+  - Google Play services
+  - Intel x86 Emulator Accelerator (HAXM installer)
+  - Support Repository
+    - Android Support Repository
+    - Google Repository
+
+  ![image](sdk_tools.png)
 
 - SDK Platforms (you may have to click "Show Package Details" to expand packages)
-    ![image](/img/mobile/sdk_platforms.png)
-    - Android 7 (Nougat) or above ({{< newtabref href="https://github.com/mattermost/mattermost-mobile/issues/2480" title="We've dropped Android 5/6 Support since December 2018, you may still continue to use 1.14 for Android 5/6 devices" >}})
-        - Google APIs
-        - SDK Platform
-            - For Android Q or above > Android SDK Platform 29 or above
-        - Intel or Google Play Intel x86 Atom\_64 System Image
-    - Any other API version that you want to test
+  - Android 7 (Nougat) or above
+    - Google APIs
+    - SDK Platform
+      - For Android Q or above > Android SDK Platform 29 or above
+    - Intel or Google Play Intel x86 Atom\_64 System Image
+  - Any other API version that you want to test
+
+  ![image](sdk_platforms.png)
+
+{{<note "Android 5/6 support">}}
+We've dropped Android 5/6 Support since December 2018; you may still continue to use 1.14 for Android 5/6 devices. See the following GitHub issue for more information: {{<newtabref href="https://github.com/mattermost/mattermost-mobile/issues/2480" title="Android 5/6 support">}}
+{{</note>}}
 
 ## Obtain the source code
 
 In order to develop and build the Mattermost mobile apps, you'll need to get a copy of the source code. Forking the `mattermost-mobile` repository will also make it easy to contribute your work back to the project in the future.
 
-1.  Fork the {{< newtabref href="https://github.com/mattermost/mattermost-mobile" title="mattermost-mobile" >}} repository on GitHub.
+1. Fork the {{< newtabref href="https://github.com/mattermost/mattermost-mobile" title="mattermost-mobile" >}} repository on GitHub.
 
 2. Clone your fork locally:
-    - Open a terminal
-    - Change to a directory you want to hold your local copy
-    - Run `git clone https://github.com/<username>/mattermost-mobile.git` if you want to use HTTPS, or `git clone git@github.com:<username>/mattermost-mobile.git` if you want to use SSH
 
-    **`<username>` refers to the username or organization in GitHub that forked the repository**
+   - Open a terminal
```

Comment:
Please convert these bulleted items with a numbered list of sub-steps (a., b., c.,) so that users know to perform each step in the order prescribed.